test/Conversion/LinalgToTpp/linalg-to-tpp-memref-unsupported.mlir

```diff
@@ -6,7 +6,7 @@
 // CHECK-SAME: %[[arg2:.*]]: memref<5x5xf32>) {
 func.func @brgemm_lowering(%arg0: memref<3x5x4xf32>, %arg1: memref<3x4x5xf32>,
                           %arg2: memref<5x5xf32>) {
-  // CHECK: tpp.brgemm ins(%[[arg0]] : memref<3x5x4xf32>, %[[arg1]] : memref<3x4x5xf32>, %[[arg2]] : memref<5x5xf32>) outs(%[[arg2]] : memref<5x5xf32>)
+  // CHECK-NOT: tpp.brgemm ins(%[[arg0]] : memref<3x5x4xf32>, %[[arg1]] : memref<3x4x5xf32>, %[[arg2]] : memref<5x5xf32>) outs(%[[arg2]] : memref<5x5xf32>)
```

Comment:
IIUC, most of the tests here (non-matching) have been moved to `linalg-to-tpp-on-tensors.mlir` and are not just redundant, but wrong. We don't match at memrefs any more not because the shape is wrong, but because the type is wrong.

I recommend to remove every test here that has been moved to the tensor file and make sure we only leave tests here (one per op) to make sure we don't match "valid shapes" to TPP on memrefs.

Reply:
Good idea, cleaned up the memref.
All cases apart from memref stride specific tests were moved to `linalg-to-tpp-tensor` test file.